src/orion/core/io/database/pickleddb.py

```diff
@@ -8,20 +8,25 @@
    :synopsis: Implement permanent version of :class:`orion.core.io.database.EphemeralDB`
 
 """
+
 from contextlib import contextmanager
+import logging
 import os
 import pickle
+from pickle import PicklingError
 
 from filelock import FileLock
 
 import orion.core
 from orion.core.io.database import AbstractDB
 from orion.core.io.database.ephemeraldb import EphemeralDB
 
+log = logging.getLogger(__name__)
 
 DEFAULT_HOST = os.path.join(orion.core.DIRS.user_data_dir, 'orion', 'orion_db.pkl')
 
 
+# pylint: disable=protected-access
```

Comment:
This could be moved to the only access to `database._db` in `_find_unpickable_doc`